testdata/p4_16_samples_outputs/psa-unicast-or-drop-corrected-bmv2.p4.spec

```diff
@@ -44,10 +44,12 @@ apply {
 	cast  h.ethernet.dstAddr bit_32 m.psa_ingress_output_metadata_egress_port
 	jmpneq LABEL_0END h.ethernet.dstAddr 0x0
 	mov m.psa_ingress_output_metadata_drop 1
-	LABEL_0END :	emit h.ethernet
+	LABEL_0END :	jmpneq LABEL_DROP m.psa_ingress_output_metadata_drop 0x0
+	emit h.ethernet
 	extract h.ethernet
 	emit h.ethernet
 	tx m.psa_ingress_output_metadata_egress_port
```

Comment:
Is the behavior of the `tx` primitive such that execution stops there?  In other words, what prevents `tx` from executing, and then falling through to execute the `drop` statement below?

Reply:
yes, the processing will terminate after `tx`.